p4include/v1model.p4

```diff
@@ -250,8 +277,31 @@ extern action_profile {
  */
 extern void random<T>(out T result, in T lo, in T hi);
 
-// If the type T is a named struct, the name is used
-// to generate the control-plane API.
+/***
+ * Calling digest causes a message containing the values specified in
+ * the data parameter to be sent to the control plane software.  It is
+ * similar to sending a clone of the packet to the control plane
+ * software, except that it can be more efficient because the messages
+ * are typically smaller than packets, and many such small digest
+ * messages are typically coalesced together into a larger "batch"
+ * which the control plane software processes all at once.
+ *
+ * Calling digest is only supported in the ingress control.  There is
+ * no way to undo its effects once it has been called.
+ *
+ * If the type T is a named struct, the name is used to generate the
+ * control plane API.
+ *
+ * TBD: Document how the value receiver is used.  Does the BMv2
+ * implementation support more than one value of receiver in the same
```

Comment:
the `receiver` parameter is ignored, the digest is processed according to the runtime interface being used